fitlins/interfaces/bids.py

```diff
@@ -211,24 +213,9 @@ class LoadBIDSModel(SimpleInterface):
     def _run_interface(self, runtime):
         from bids.analysis import Analysis
         from bids.layout import BIDSLayout
-        import re
-
-        force_index = [
```

Comment:
wait, I think this is important, because of this:  "If entry looks like `/<pattern>/`, treat `<pattern>` as a regex"

I think pybids doesn't respect the `/` so we need to strip those and preprocess CLI arguments.

Reply:
Yeah, I think we need to move this to `cli.py`.

Reply:
Agreed